packages/odyssey-react-mui/src/icons.generated/Aura.tsx

```diff
@@ -0,0 +1,45 @@
+/*!
```

Comment:
not sure if we already have these icons somewhere.

Reply:
The icons should be in Figma. we then run a script to pull them into Odyssey.

If they're not in Figma, then when we run the "pull down new icons" script, it will remove this one.

There are other scripts we run to process the icons too. Then we make a PR and get those up.